src/textual/widgets/_data_table.py

```diff
@@ -1116,10 +1116,9 @@ def move_cursor(
             cursor_row = row
         if column is not None:
             cursor_column = column
-
         destination = Coordinate(cursor_row, cursor_column)
         self.cursor_coordinate = destination
-        self.call_after_refresh(self._scroll_cursor_into_view, animate=animate)
+        self._scroll_cursor_into_view(animate=animate)
```

Comment:
It's good to get rid of the `call_after_refresh`, but I do wonder what it was solving originally.

In hindsight I wish we had added a comment there. Might be a good policy going forward every time we use `call_after_refresh` as a workaround.

Reply:
I actually added the `call_after_refresh` here last week because it was solving the problem of the DataTable not scrolling if you added a bunch of rows and then immediately moved the cursor.

The `call_after_refresh` is required to ensure the DataTable only scrolls after the virtual height settles inside on_idle.

The removal here was a merge conflict issue - I was getting this confused with another piece of code where I actually did remove the `call_after_refresh` - good spot!